GNOME #436805; Debian #287725; correct PostLogin sample file comments: the name
of the environment variable for the login name is $LOGNAME, not $LOGIN

--- gdm-2.16.4.orig/config/PostLogin
+++ gdm-2.16.4/config/PostLogin
@@ -4,5 +4,5 @@
 # file to <gdmconfdir>/PostLogin/Default for this script to be run.  This
 # script will be run before any setup is run on behalf of the user and is
 # useful if you for example need to do some setup to create a home directory
-# for the user or something like that.  $HOME, $LOGIN and such will all be
+# for the user or something like that.  $HOME, $LOGNAME and such will all be
 # set appropriately and this script is run as root.
